src/lib/models/automatic-speech-recognition/whisper-base-en.ts

```diff
@@ -1,9 +1,10 @@
 import {AutomaticSpeechRecognitionBaseModel} from '@/lib/models/base';
+import {ClientConfig} from "@/lib/types/common/client-config";
 
 export class WhisperBaseEn extends AutomaticSpeechRecognitionBaseModel {
   public static readonly endpoint = 'https://api.deepinfra.com/v1/inference/openai/whisper-base.en';
 
-  constructor(authToken: string) {
-    super(WhisperBaseEn.endpoint, authToken);
+  constructor(authToken: string, config?: ClientConfig) {
```

Comment:
I guess it makes sense to pass only some of the config properties when you call, the rest should be defaulted. Not sure if a separate class with optional fields makes more sense, or just use the optional fields everywhere, just assume non-null inside the client code (because we won't let them be null there).